ligo/haskell/test/Test/Ligo/RegistryDAO.hs

```diff
@@ -191,6 +191,51 @@ test_RegistryDAO =
               -- Now we expect this to work
               withSender (AddressResolved wallet1) $
                 call baseDao (Call @"Propose") (ProposeParams requiredFrozen largeProposalMeta)
+
+    , nettestScenarioCaps "checks on-chain view correctly returns the registry value" $ do
+        let frozen_scale_value = 1
+        let frozen_extra_value = 0
+        let slash_scale_value = 1
+        let slash_division_value = 1
+        withOriginated 3
+          (\(admin: wallet1: voter1:_) -> setExtra @Natural [mt|slash_division_value|] 2 $
+            setExtra @Natural [mt|frozen_scale_value|] frozen_scale_value $
+            setExtra @Natural [mt|frozen_extra_value|] frozen_extra_value $
+            setExtra @Natural [mt|slash_scale_value|] slash_scale_value $
+            setExtra @Natural [mt|max_proposal_size|] 200 $
+            setExtra @Natural [mt|slash_division_value|] slash_division_value $ initialStorageWithExplictRegistryDAOConfig admin [wallet1, voter1]) $
+
+          \(admin: wallet1: voter1 : _) _ baseDao -> let
+            proposalMeta = DynamicRec $ Map.fromList $
+              [ ([mt|updates|], lPackValueRaw [([mt|key|], Just [mt|testVal|])])
+              , ([mt|agoraPostID|], lPackValueRaw @Natural 10)
+              ]
+            proposalSize = metadataSize proposalMeta
+
+            in do
+              runIO $ putTextLn $ show proposalSize
+              let requiredFrozen = proposalSize * frozen_scale_value + frozen_extra_value
```

Comment:
Since `frozen_scale_value = 1` and `frozen_extra_value = 0`, you may as well skip these calculations.

Also, a nitpick: I think it would be a bit better if you started this `do` block at line 208, so that the test has one less level of indentation.

Reply:
Changed.